src/css/index.css

```diff
@@ -189,7 +189,7 @@ a.site {
     overflow: visible;
 
     &::before {
-      content: "or";
+      content: attr(divider-text);;
```

Comment:
https://accessibleweb.com/question-answer/how-is-css-pseudo-content-treated-by-screen-readers/

may be nitpicking but this probably shouldn't be a pseudo element for accessibility reasons

Reply:
https://guides.github.com/activities/hello-world/readme-edits.gif

Reply:
Yeah never hurts to make sure the built-in pages are accessibility-friendly. definitely not in scope of this PR - we should probably open a new issue to track this.